src/ActionType.tsx

```diff
@@ -0,0 +1,50 @@
+/** Semantic type of action event to log. */
+export enum ActionType {
```

Comment:
side-comment - eventually, we'll auto-generate the typescript output to a public repo so they can be used.  not urgent.

Reply:
Great!